extensions-contrib/azure-extensions/src/main/java/io/druid/storage/azure/AzureDataSegmentPuller.java

```diff
@@ -38,6 +38,8 @@
 {
   private static final Logger log = new Logger(AzureDataSegmentPuller.class);
 
+  String HADOOP_BLOB_PATH_IDENTIFIER = "blob.core.windows.net";
```

Comment:
`static final`

Reply:
From where does this constant come? Is it specified somewhere in Azure docs? Please add a Javadoc comment